src/expertsystem/reaction/particle.py

```diff
@@ -684,19 +637,19 @@ def __create_isospin(pdg_particle: PdgDatabase) -> Optional[Spin]:
     if pdg_particle.I is None:
         return None
     magnitude = pdg_particle.I
-    projection = __compute_isospin_projection(pdg_particle)
+    projection = __isospin_projection_from_pdg(pdg_particle)
     return Spin(magnitude, projection)
 
 
-def __compute_isospin_projection(pdg_particle: PdgDatabase) -> float:
+def __isospin_projection_from_pdg(pdg_particle: PdgDatabase) -> float:
     if pdg_particle.charge is None:
         raise ValueError(f"PDG instance has no charge:\n{pdg_particle}")
     if "qq" in pdg_particle.quarks.lower():
         strangeness, charmness, bottomness, topness = __compute_quark_numbers(
             pdg_particle
         )
         baryon_number = __compute_baryonnumber(pdg_particle)
-        projection = GellmannNishijima.compute_isospin_projection(
+        projection = _compute_isospin_projection(
```

Comment:
I would use the above line here directly
```charge - 0.5 * (
￼        baryon_number + strangeness + charmness + bottomness + topness
￼    )
```

Reply:
3d13ff8